packages/ketcher-react/src/script/ui/views/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,133 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+import React, { useRef, useEffect } from 'react'
+import * as KN from 'w3c-keyname'
+import clsx from 'clsx'
+
+import style from './Dialog.module.less'
+
+interface DialogProps {
+  children: React.ReactElement
+  title: string
+  params: DialogParams
+  result: () => any
+  valid: () => boolean
+  buttons: Array<string | React.ReactElement>
+  className: string
+}
+interface DialogParams extends DialogCallProps {
+  className: string
+}
```

Comment:
I don't understand this interface, why it is? 
`DialogCallProps` is function-props, and `DialogProps` is all another props.
`className` is not a function, so it should be in `DialogProps`, if `className` should be passed directly in container component, it should be described **in the** container. Component should know nothing about it

Reply:
I've created `DialogParams` to decompose `params` property in the `DialogProps` because `params` consists of function-props and other props.